docs/rules/index.md

```diff
@@ -45,21 +45,55 @@ The `--fix` option on the [command line](../user-guide/command-line-interface#fi
 
 {{ site.data.rules.deprecated.description }}
 
-<div class="deprecated-rules" markdown="1">
-|Deprecated rule|Replaced by
-|---|---
-{% for rule in site.data.rules.deprecated.rules %}|[{{rule.name}}]({{rule.name}})| {% for replaced in rule.replacedBy %} [{{replaced}}]({{replaced}}) {% endfor %}
+<div class="rule-list deprecated-rules">
+<table class="table table-striped">
+
```

Comment:
Could you add colgroup, so we could make both removed and deprecated tables align?